docs/topics/wildfly-swarm/docs/reference/fractions/keycloak.adoc

```diff
@@ -177,4 +204,115 @@ The session id is changed by default on a successful login.  Change this to true
 swarm.keycloak.secure-deployments._KEY_.use-resource-role-mappings:: 
 Use resource level permissions from token
 
+swarm.keycloak.secure-servers._KEY_.allow-any-hostname:: 
+SSL Setting
+
+swarm.keycloak.secure-servers._KEY_.always-refresh-token:: 
+Refresh token on every single web request
+
+swarm.keycloak.secure-servers._KEY_.auth-server-url:: 
+Base URL of the Realm Auth Server
+
+swarm.keycloak.secure-servers._KEY_.auth-server-url-for-backend-requests:: 
+URL to use to make background calls to auth server
+
+swarm.keycloak.secure-servers._KEY_.autodetect-bearer-only:: 
+autodetect bearer-only requests
+
+swarm.keycloak.secure-servers._KEY_.bearer-only:: 
+Bearer Token Auth only
+
+swarm.keycloak.secure-servers._KEY_.client-key-password:: 
+n/a
+
+swarm.keycloak.secure-servers._KEY_.client-keystore:: 
+n/a
+
+swarm.keycloak.secure-servers._KEY_.client-keystore-password:: 
+n/a
+
+swarm.keycloak.secure-servers._KEY_.confidential-port:: 
+Specify the confidential port (SSL/TLS) used by the Realm Auth Server
+
+swarm.keycloak.secure-servers._KEY_.connection-pool-size:: 
+Connection pool size for the client used by the adapter
+
+swarm.keycloak.secure-servers._KEY_.cors-allowed-headers:: 
+CORS allowed headers
+
+swarm.keycloak.secure-servers._KEY_.cors-allowed-methods:: 
+CORS allowed methods
+
+swarm.keycloak.secure-servers._KEY_.cors-exposed-headers:: 
+CORS exposed headers
+
+swarm.keycloak.secure-servers._KEY_.cors-max-age:: 
+CORS max-age header
+
+swarm.keycloak.secure-servers._KEY_.credentials._KEY_.value:: 
+Credential value
+
+swarm.keycloak.secure-servers._KEY_.disable-trust-manager:: 
+Adapter will not use a trust manager when making adapter HTTPS requests
+
+swarm.keycloak.secure-servers._KEY_.enable-basic-auth:: 
+Enable Basic Authentication
+
+swarm.keycloak.secure-servers._KEY_.enable-cors:: 
+Enable Keycloak CORS support
+
+swarm.keycloak.secure-servers._KEY_.expose-token:: 
+Enable secure URL that exposes access token
+
+swarm.keycloak.secure-servers._KEY_.ignore-oauth-query-parameter:: 
+disable query parameter parsing for access_token
+
+swarm.keycloak.secure-servers._KEY_.min-time-between-jwks-requests:: 
+If adapter recognize token signed by unknown public key, it will try to download new public key from keycloak server. However it won't try to download if already tried it in less than 'min-time-between-jwks-requests' seconds
```

Comment:
single quoted value needs to use backticks

Reply:
Note to self: not easily fixable in our sources.